octodns/provider/route53.py

```diff
@@ -551,27 +931,44 @@ def _healthcheck_measure_latency(self, record):
             .get('measure_latency', True)
 
     def _health_check_equivilent(self, host, path, protocol, port,
-                                 measure_latency, health_check,
-                                 first_value=None):
+                                 measure_latency, health_check, value=None):
         config = health_check['HealthCheckConfig']
+
+        # So interestingly Route53 normalizes IPAddress which will cause us to
+        # fail to find see things as equivalent. To work around this we'll
+        # ip_address's returned object for equivalence
+        # E.g 2001:4860:4860::8842 -> 2001:4860:4860:0:0:0:0:8842
```

Comment:
This would have been a bug with `AAAA` geo records as well. Guess no one is using them or if they are they're fully specified.